### DISCUSSION TITLE
don't work in the KernelSU

### DISCUSSION THREAD

**Discussion by alinnse** [mywalkb/LSPosed_mod#17] [General]
![Screenshot_2023-07-21-12-06-32-45_9e8df3d0c7c1f50248b6ee043a653d26.jpg](https://github.com/mywalkb/LSPosed_mod/assets/23187434/47410ad1-1733-41d3-aac9-23f948c8ee50)

**Comment by mywalkb** [comment]
I don't have a kernelsu device but should be easy change the script cli for work without magisk.
Do you know in which path are installed modules?

**Comment by mywalkb** [comment]
Could you try this version? if works I will commit.

[cli.txt](https://github.com/mywalkb/LSPosed_mod/files/12123528/cli.txt)

**Comment by mywalkb** [comment]
Could you test [this](https://github.com/mywalkb/LSPosed_mod/actions/runs/5626289389) action?
It include the fix for kernelsu and latest patch from original repo.

**Reply by alinnse** [reply]
Sorry, my phone has been sent for after-sales repair and it will take some time for testing. Thank you very much

**Comment by alinnse** [comment]
> Could you test [this](https://github.com/mywalkb/LSPosed_mod/actions/runs/5626289389) action?
> It include the fix for kernelsu and latest patch from original repo.
> 

hi,after replacing the file, test it and it will work. Check whether the LSPosed module is enabled/disabled normally.Thank you again!
![Screenshot_2023-07-23-23-58-26-79_9e8df3d0c7c1f50248b6ee043a653d26.jpg](https://github.com/mywalkb/LSPosed_mod/assets/23187434/e6bc5fca-05c6-459b-b6e1-0926df1140c9)
